src/transformers/models/ict/image_processing_ict.py

```diff
@@ -0,0 +1,320 @@
+# coding=utf-8
+# Copyright 2023 Authors at City University of Hong Kong, Microsoft Cloud + AI,
+# The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Image processor class for ICT."""
+
```

Comment:
I assume what to write in the copyright message would change depending on what the authors wanted to be called as and the license they end up using, right?

Reply:
Yes, exactly, you can change it based on what the authors want